src/cctag/utils/LogTime.cpp

```diff
@@ -17,7 +17,7 @@ bool Mgmt::Measurement::doPrint( ) const
 
 void Mgmt::Measurement::print( std::ostream& ostr ) const
 {
-    if( not _probe ) return;
```

Comment:
It's not that windows doesn't support "not", it's that gcc and clang are too lax. You must include `<iso646.h>` to get and/not/etc. "keywords".